Makefile

```diff
@@ -13,6 +15,13 @@ lint:
 test_rules:
 	./automation/promtool_tests.sh test_rules
 
-.PHONY: pint
 pint:
+	echo 'Installing pint...'
+	curl -OJL 'https://github.com/cloudflare/pint/releases/download/v${PINT_VERSION}/pint-${PINT_VERSION}-linux-amd64.tar.gz'
+	tar xvzf 'pint-${PINT_VERSION}-linux-amd64.tar.gz' pint-linux-amd64
+	mv pint-linux-amd64 pint
+	rm 'pint-${PINT_VERSION}-linux-amd64.tar.gz'	
+	
+.PHONY: pint_lint
+pint_lint:	
```

Comment:
You need to modify the call to make pull-request.yaml so it refer to this target. Currently, it just runs the `pint` target again.
L97 there should be:
```
script: yum install -y make && make pint_lint
```